connPFM/tests/test_ev.py

```diff
@@ -39,23 +39,15 @@ def test_threshold_ets_matrix():
     dum_mat[2, 1] = 3
     dum_mat2 = np.zeros((3, 3))
     dum_mat2[2, 1] = 3
-    th_dum = ev.threshold_ets_matrix(dum_mat, [2], 2)
+    th_dum = utils.threshold_ets_matrix(dum_mat, thr=2, selected_idxs=2)
     assert np.all(th_dum == dum_mat2)
 
 
 def test_event_detection(
```

Comment:
You should add the different rss options to the unittest testing so we don't lose coverage

Reply:
I'm planning on it. That's why this is still not ready for a review 😝